src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEnum.cs

```diff
@@ -146,6 +146,21 @@ public enum AccessibilityState
     /// <since_tizen> 3 </since_tizen>
     public partial class View
     {
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Please add a summary for this method with the role of parameters.

Reply:
I propose change of "v" parameter to "state" and addition of the following description :

        /// <summary>
        /// A helper method to manipulate individual bit flags (e.g. turn them on or off)
        /// </summary>
        /// <param name="obj">An object that accumulates combination of bit flags</param>
        /// <param name="bit">A bit flag to be operated</param>
        /// <param name="state">A state of the bit flag to be set (0 == off, 1 ==on)</param>